src/cli/cli.cpp

```diff
@@ -4984,7 +4984,8 @@ template <> otError Interpreter::Process<Cmd("neighbor")>(Arg aArgs[])
                 OutputFormat("|%1d", neighborInfo.mFullNetworkData);
                 OutputFormat("| ");
                 OutputExtAddress(neighborInfo.mExtAddress);
-                OutputLine(" |");
+                OutputFormat(" | %7d |", neighborInfo.mVersion);
+                OutputNewLine();
```

Comment:
I think we can combine two call into `OutputLine()`?
```suggestion
                OutputLine(" | %7d |", neighborInfo.mVersion);
```

Reply:
Done.